tini/src/seccomp_fd_notify.c

```diff
@@ -161,7 +155,18 @@ struct sock_filter net_filter[] = {
 		/* Trap connect */
 		BPF_JUMP(BPF_JMP | BPF_JEQ | BPF_K, __NR_connect, 0, 1),
 		BPF_STMT(BPF_RET + BPF_K, SECCOMP_RET_USER_NOTIF),
+	};
 
+	struct sock_filter traffic_steering_filter[] = {
```

Comment:
Does it make sense to borrow the cBPF code from the kernel which rewrites jumps?

Reply:
Looked at https://github.com/torvalds/linux/blob/master/samples/seccomp/bpf-fancy.c#L44. We can borrow that. Need to spend some time on crafting that. Will open another PR.